nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-ui/src/main/resources/META-INF/NOTICE

```diff
@@ -1,8 +1,17 @@
 Apache NiFi
 Copyright 2014-2021 The Apache Software Foundation
+
 This product includes software developed at
 The Apache Software Foundation (http://www.apache.org/).
 
+******************
+Apache Software License v2
+******************
+The following binary components are provided under the Apache Software License v2
+  (ASLv2) Carbon Design System Icons
+    The following NOTICE information applies:
+      Copyright 2015 IBM Corp.
+
```

Comment:
Is this addition still necessary with the change to flowfont?

Reply:
Good question. I took the icons I needed from the Carbon Design System Icons and I integrated them into Flowfont using Fontello. Fontello was used 5 years ago to generate Flowfont: https://fontello.com/

So technically, I used some components from Carbon Design System Icons. As I understand (but I'm not a lawyer), I should add the license of the Carbon Design System Icons.

If you look at the main license file, we reference the Fontello license.  https://github.com/apache/nifi/blob/7fe8858d22f580ad9dcd96ebafb59dbe5e73f5cb/nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-ui/src/main/resources/META-INF/LICENSE#L681-L701

As we reference the tool used to generate the font, I think we should reference the license of the icons used in the app. 

Or we need to think about removing the reference to the Fontello license.

What do you think?

Reply:
Thanks for the thorough response and description.  I am not an expert when it comes to licensing, but since you clarified that the Flowfont file includes icons from the Carbon Design System, I think the updated notice you included makes sense.